src/tokens/token-groups/typography.json

```diff
@@ -1,6 +1,31 @@
 {
+  "font-family-base": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
+  "font-family-mono": "ui-monospace, SFMono-Regular, 'SF Mono', Consolas, 'Liberation Mono', Menlo, monospace",
```

Comment:
What about `font-family-sans` and `font-family-mono`? This makes more sense to me.

Reply:
Makes sense to me as well 👍 Looks like [open props](https://github.com/argyleink/open-props/blob/b16fe44169cb76f5d2340e960fa25233254b5f41/src/props.fonts.css#L2-L4) takes a similar approach, though I prefer the explicit naming in your suggestions.